src/typedefs.jl

```diff
@@ -29,28 +42,48 @@ end
 
 ## Values
 
-struct CategoricalValue{T, R <: Integer} <: AbstractString
+"""
+Default "categorical value" type for
```

Comment:
Could you remove quotes around mentions of "categorical value"? I don't find it very useful, and for example in the docstring below they are not present.